From 976b7c864942733731dbee8c09048b02a91b8a1e Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Fri, 19 Dec 2008 18:20:11 -0800
Subject: Staging: android: remove dummy android.c driver

From: Greg Kroah-Hartman <gregkh@suse.de>

There are files now in the drivers/staging/android/ directory, so the
dummy android.c file can be safely removed.

Cc: Robert Love <rlove@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/android/Makefile  |    1 -
 drivers/staging/android/android.c |   18 ------------------
 2 files changed, 19 deletions(-)
 delete mode 100644 drivers/staging/android/android.c

--- a/drivers/staging/android/android.c
+++ /dev/null
@@ -1,18 +0,0 @@
-#include <linux/kernel.h>
-#include <linux/init.h>
-#include <linux/module.h>
-
-static int __init android_init(void)
-{
-	return 0;
-}
-
-static void __exit android_exit(void)
-{
-}
-
-module_init(android_init);
-module_exit(android_exit);
-
-MODULE_AUTHOR("Greg Kroah-Hartman");
-MODULE_LICENSE("GPL");
--- a/drivers/staging/android/Makefile
+++ b/drivers/staging/android/Makefile
@@ -1,4 +1,3 @@
-obj-$(CONFIG_ANDROID)			+= android.o
 obj-$(CONFIG_ANDROID_BINDER_IPC)	+= binder.o
 obj-$(CONFIG_ANDROID_LOGGER)		+= logger.o
 obj-$(CONFIG_ANDROID_RAM_CONSOLE)	+= ram_console.o
